client/packages/common/src/hooks/useDialog/useDialog.tsx

```diff
@@ -154,13 +160,17 @@ export const useDialog = (dialogProps?: DialogProps): DialogState => {
     enableAutocomplete,
     sx = {},
     deleteButton,
+    fullWidthOnMobile,
   }) => {
     // The slide animation is triggered by cloning the next button and wrapping the passed
     // on click with a trigger to slide.
     const { slideConfig, onTriggerSlide } = useSlideAnimation(
       isRtl,
       animationTimeout
     );
+    const { isOpen: keyboardIsOpen } = useKeyboardContext();
```

Comment:
NIT: Noticed that sometimes the context values are destructured and sometimes they're not. Not sure where we stand on things. I'm a big fan of destructuring, but I'm wondering if we should update the variables returned from the context to `isKeyboardOpen` and `isKeyboardEnabled`. This would remove the need to alias them as well 😃